lib/order.rb

```diff
@@ -0,0 +1,33 @@
+require './lib/menu.rb'
+
+class Order
+
+attr_reader :list_of_dishes, :selected_items
+
+def initialize(m = Menu.new)
+  @list_of_dishes = m
+  @selected_items = []
+end
+
+def view_dishes
+  @item = @list_of_dishes.list
```

Comment:
Is it necessary to do a variable assignment here and create this new @item instance variable?

It seems to me like you could achieve the same thing using
```ruby
def view_dishes
  list_of_dishes.list
end
```